modules/rosa-accessing-your-cluster.adoc

```diff
@@ -4,9 +4,14 @@
 
 
 [id="rosa-accessing-your-cluster"]
-= Accessing your cluster
+= Accessing your cluster with an IDP account
 
-To log in to your cluster, you must configure an identity provider (IDP). This procedure uses GitHub as an example IDP. To view other supported IDPs, run `rosa create idp --help`.
+To log in to your cluster, you can configure an identity provider (IDP). This procedure uses GitHub as an example IDP. To view other supported IDPs, run `rosa create idp --help`.
```

Comment:
In IBM Style Guide (ISG) they're following the command with _command_.

> To view other supported IDPs, run the `rosa create idp --help` command.

Reply:
Done.